### DISCUSSION TITLE
v4.3.1 GTK, can't resize Audacious since upgrading to Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by djackson404** [audacious-media-player/audacious#1392] [General]
Additionally: if I maximize Audacious, I can't un-maximize it.
Had to switch to the Winamp style interface just so I could reasonably use Audacious.

I have more than one system updated to Ubuntu 24.04, and they all have the same problem.

**Comment by djackson404** [comment]
Update: I noticed the same exact problem with GoldenCheetah. Might need to downgrade back to Ubuntu 23.10.

**Comment by radioactiveman** [comment]
No need to downgrade. You only need to use a X11 session instead of Wayland (can be selected in the login screen) or start Audacious via XWayland.

For Qt: `QT_QPA_PLATFORM=xcb audacious`
For GTK: `GDK_BACKEND=x11 audacious --gtk`

We will make this more comfortable with Audacious 4.4.

Oh, and please search other conversations here. This has been reported multiple times already. :)

**Reply by djackson404** [reply]
Good to know, thank you.
I did search at first but I guess I didn't use the right search terms.
For what it's worth on my HTPC (one of the three computers I have that I upgraded) I did a 100% clean install of 24.04, and the problems don't appear to exist now, so I'm suspecting there's something funny about the upgrade process that's causing this. Guess it'll be a while before the actual cause is uncovered.
